metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -408,14 +431,14 @@ def __init__(
     ):
         super().__init__(config, ctx)
 
-        self.config = config
-        self.report = StaleEntityRemovalSourceReport()
-        self.server = None
-        self.upstream_tables = {}
```

Comment:
removed `upstream_tables` variable. Replaced with `database_tables` which is better typed.